### PR TITLE
Add message after log rotation

### PR DESCRIPTION
**Pull request by Alkarex**
https://github.com/FreshRSS/FreshRSS/pull/1712
https://github.com/FreshRSS/FreshRSS/issues/1562

### CODE REVIEW

File: lib/Minz/Log.php [hunk 1]
```diff
@@ -98,6 +98,7 @@ protected static function ensureMaxLogSize($file_name) {
 				rewind($fp);
 				ftruncate($fp, 0);
 				fwrite($fp, $content ? $content : '');
+				fwrite($fp, '[' . date('r') . "] [notice] --- Log rotate.\n";);
```
Comment (aledeg) [inline lib/Minz/Log.php]: I am not a big fan of string concatenation. I'd like ```sprintf``` better. My 2¢

Comment (Frenzie) [reply lib/Minz/Log.php]: Is PHP one of the languages where it makes a difference in speed? In this case it'd make no difference in legibility but if you end up with `%s%s%s` then concat can still be clearer.

(Ideally I suppose you'd have something more like Python's string.format instead.)

Comment (Alkarex) [reply lib/Minz/Log.php]: Updated to `sprintf()` for @aledeg :-)

Comment (aledeg) [reply lib/Minz/Log.php]: @Frenzie @Alkarex it makes a difference on how the memory is allocated. String concatenation use more memory than sprintf.

Comment (Frenzie) [reply lib/Minz/Log.php]: @aledeg In that case I too prefer sprintf, although without seeing benchmarks it might be premature optimization. ;-D

Comment (aledeg) [reply lib/Minz/Log.php]: @Frenzie I know that from one of the PHP core dev. He made a speak about optimization and that was one of it.

**Pull request merged**